module2/homework/grayscaleImages/main.cpp

```diff
@@ -0,0 +1,52 @@
+#include "compression.hpp"
+
+#include <array>
+#include <forward_list>
+#include <cstdint>
```

Comment:
array and cstdint inclusion not needed here

Reply:
+1